mailpoet/assets/css/src/components-automation-listing/_listing.scss

```diff
@@ -1,3 +1,34 @@
 #mailpoet_automation * {
   box-sizing: border-box;
+
+  .mailpoet-add-new-button {
+    padding-right: 12px;
+  }
+
+  .mailpoet-automation-listing {
+    /* Prevent border radius beneath tabs */
+    border-radius: 0 0 1px 1px;
+    /* Prevent overlapping box shadow since listing is rendered inside tabs */
+    clip-path: inset(-1px 0 0 0);
```

Comment:
I wonder whether it's a good idea to clip out the `box-shadow` rather than resetting the box shadow itself. Or maybe I'm not understanding what exactly this does. Can the listing have all borders and box shadows reset and can we apply them only on the outer tabs, maybe?

Anyway, the result looks good 👍 So maybe no need to fix anything here.

Reply:
The intention here is to clip out all of the box shadow except for 1px of shadow on the top.

It would be possible to use a border instead of a box shadow, but then I'm not sure if there's an elegant way to get the bottom border of the active tab to cover the border.

Using border:
<img width="424" alt="image" src="https://user-images.githubusercontent.com/8656640/188710977-1eec236e-1968-4638-847c-e89c23939caf.png">

Using box shadow with clipping:
<img width="493" alt="image" src="https://user-images.githubusercontent.com/8656640/188710906-daa247f1-71fb-4562-af08-12f07a4d8113.png">

I think it actually makes a pretty big difference zoomed out (this is just me switching between the two versions a few times):

https://user-images.githubusercontent.com/8656640/188711373-b6d5e405-60be-492d-b963-d428847ebe60.mov

I think I'll just leave this as-is for now and hopefully we can come up with a cleaner solution in the future when we refactor this into a proper component.